tools/block-generator/scenarios/benchmarks/stress.25000.yml

```diff
@@ -0,0 +1,32 @@
+name: "Stress (25000)"
+genesis_accounts: 10000
+genesis_account_balance: 1000000000000
+tx_per_block: 25000
+
+# transaction distribution
+tx_pay_fraction: 0.05
+tx_app_fraction: 0.95
+
+# payment config
+pay_acct_create_fraction: 1.00
+pay_xfer_fraction: 0.00
+
+# asset config
+asset_create_fraction: 0.001
+asset_optin_fraction: 0.1
+asset_close_fraction: 0.05
+asset_xfer_fraction: 0.849
+asset_delete_fraction: 0
+
+# app kind config
+app_boxes_fraction: 1.0
+app_swap_fraction: 0.0
+
+# app boxes config
+app_boxes_create_fraction: 0.5
+app_boxes_optin_fraction: 0.5
+app_boxes_call_fraction: 0.0
+
+# app swap config
```

Comment:
These can be omitted (here and elsewhere where no swaps are happening), but it's ok to leave in.